target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -72,6 +72,11 @@ ramips_setup_interfaces()
 		uci add_list firewall.@zone[1].network='eth_data'
 		uci add_list firewall.@zone[1].network='eth_om'
 		;;
+	cudy,m1800|\
+	yuncore,ax820|\
+	zyxel,nt7101)
+		ucidef_set_interfaces_lan_wan "lan" "wan"
+		;;	;;
```

Comment:
Wrong here. And port this feature may helpful. f1c9afd801380a05a91d979b475c76cc0a67caae

Reply:
That's a cool thing - thanks for pointing it out!
The unused CPU ports always bothered me a little :-D

Reply:
i've implemented the change but haven't tested it yet.
hopefully, later today.

Reply:
We need to mux phy4 (lan port) to gmac1 in this device, not phy3 (wan port).

Reply:
ahh ok - understood. either 0 or 4.

Reply:
@DragonBluep i've re-arranged the ports and given the mac-increment to gmac0 to avoid both of them having the same mac address.
could you have another look?

Reply:
Everything looks fine now.

Reply:
Thanks! I'll mark this as being resolved then :-)